src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -354,4 +367,21 @@ private function normalizeValues(array $values) : array
 
        return array_values($values);
    }
+
+    private function hasJoin($queryBuilder, $alias, $association)
```

Comment:
By the way I think `getJoin` returning `null` when the join doesn't exist would be a cleaner API.